PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/ui/WebAppSettingPanel.java

```diff
@@ -496,11 +508,18 @@ public void renderWebAppsTable(@NotNull List<ResourceEx<WebApp>> webAppLists) {
                     table.setRowSelectionInterval(i, i);
                 }
             }
+            // render the deployment slot panel
```

Comment:
I think deployment slots can be rendered on selecting a web app.

Reply:
The deployment slot list is rendered on selecting a web app. But before any web app is loaded, we may disable the `Deploy to Slot` checkbox and the list. Cause it makes no sense for users to play with two items without any  web apps ready. What do you think?